src/main/java/edu/phema/elm_to_omop/translate/PhemaElmToOmopTranslator.java

```diff
@@ -69,7 +90,7 @@ public static CohortExpression generateCohortExpression(Library library, Express
 
         cohortExpression.title = expressionDef.getName();
 
-        org.ohdsi.circe.cohortdefinition.ConceptSet[] circeConceptSets = new org.ohdsi.circe.cohortdefinition.ConceptSet[conceptSets.size()];
+        ConceptSet[] circeConceptSets = new ConceptSet[conceptSets.size()];
```

Comment:
I included a few small cleanup changes like this. These explicit package qualifications were left over from the Circe Circe object conversion.